examples/c/gpio_advanced.c

```diff
@@ -16,54 +16,149 @@
 
 /* mraa header */
 #include "mraa/gpio.h"
+#include "mraa/led.h"
 
-#define GPIO_PIN 6
+/* GPIO default */
+#define GPIO_PIN1 5
+#define GPIO_PIN2 6
 
-void
-int_handler(void* args)
-{
+/* LED default value */
+#define LED_NUM1   0
+#define LED_NUM2   1
+#define LED_ON_OFF 0
+
+/* trigger type */
+#define LED_TRIGGER "heartbeat"
+
+volatile int isr_occurred;
+
+void int_handler() {
     fprintf(stdout, "ISR triggered\n");
+    isr_occurred = 1;
 }
 
-int
-main()
-{
+void isr_do(mraa_led_context led1, mraa_led_context led2, int led_on_off, int hz, int wait_idle){
```

Comment:
It's difficult for users to know what the example does.
Maybe you can add it to README.

Reply:
OK, I will make it shorter and clear.
And add it's usage to README.